client/components/tinymce/plugins/simple-payments/dialog/form.jsx

```diff
@@ -105,14 +105,14 @@ const validate = ( values, props ) => {
 					},
 				}
 			);
-			errors.price = translate( 'Price can not have more than %(countDecimal)s.', {
+			errors.price = translate( 'Price cannot have more than %(countDecimal)s.', {
 				args: { countDecimal },
 			} );
 		}
 	}
 
 	if ( ! values.email ) {
-		errors.email = translate( 'Email address can not be empty.' );
+		errors.email = translate( 'Email address cannot be empty.' );
```

Comment:
:disappointed: 24 existing translations will be lost with this change.
If the string must change, the following option has already been translated **26** times:
`translate( 'Email address should not be empty' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).